protokt-codegen/src/main/kotlin/protokt/v1/codegen/generate/MessageGenerator.kt

```diff
@@ -159,6 +159,9 @@ private class MessageGenerator(
                             .build()
                     )
                 }
+                if (property.overrides) {
```

Comment:
When the name of a field is the same as the field on the interface and we generate a non-null accessor for it, the override needs to be on the generated delegating property, not the non-null accessor.